week3/homework/src/actions/deserializeTodo.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+async function deserializeTodo(request) {
```

Comment:
👍 good job on validation. Since this function is only validating perhaps it needs a different name 😄